### ISSUE TITLE
Use consistent approach to Selenium testing across projects

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/cove-oc4ids#35]
In https://github.com/OpenDataServices/cove/blob/master/.travis.yml and elsewhere, we have a much simpler setup for Selenium. Where is [this complicated setup](https://github.com/open-contracting/cove-oc4ids/blob/master/.travis.yml) from?

@michaelwood

**Comment by michaelwood** [comment]
Mainly as I wasn't aware of the  `cove` selenium setup. The selenium setup I have used is from one I use regularly on other projects and is setup to run Chrome more or less as-is rather than ChromeHeadless which it is the default configuration used in `cove`. I have always used latest stable Chrome as  most people's Chrome browser will be "self updating" and this helps to test in a more "realistic" environment. 

The issue with chrome driver version should have had the line to keep it up to date.

```yml
- "export LATEST_CHROMEDRIVER=$(wget -q -O - http://chromedriver.storage.googleapis.com/LATEST_RELEASE)"
- "wget http://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER/chromedriver_linux64.zip; unzip chromedriver_linux64.zip -d chromedriver; export PATH=$PATH:`pwd`/chromedriver"
```
However, as you mention in the pull request #36  alternatively the chrome version could be locked and we would only need to update the chrome driver to keep it in sync with the system version, such as is needed in `cove`. A less complex setup would also then use headless version too. This would also have the advantage of consistency between projects.

**Comment by jpmckinney** [comment]
Thanks! I'll do the above for the PR. I'll rename this issue to be about consistency across projects – I had done some test commits, but for whatever reason I was getting "Chrome failed to start" though the setup was the same as other projects (as far as I can tell, the only difference being different versions of selenium, pytest, etc.).

**Comment by jpmckinney** [comment]
For reference, here was my first .travis.yml config attempt: https://travis-ci.org/open-contracting/cove-oc4ids/jobs/586822514/config

**Comment by jpmckinney** [comment]
Fixed by #62, which closed #43.
